docs/guide/limitation_reference.md

```diff
@@ -402,6 +402,25 @@ is evaluated.
 For more information on how to restrict User's access to part of the Subtree 
 follow [the example in the Admin management section](admin_panel.md#restrict-editing-to-part-of-the-tree).
 
+## Version Lock Limitation
+
+A Limitation to specify whether the User can perform edit or unlock actions on 
+Content items that are in a specific workflow stage.
+
+| | |
+|-----------------|------------------------------------------------------------------------------------------------|
+| Identifier | `VersionLock` |
+| Value Class | `Ibexa\Workflow\Security\Limitation\VersionLockLimitation.php` |
+| Type Class | `Ibexa\Workflow\Security\Limitation\VersionLockLimitationType.php` |
+| Role Limitation | yes |
+
+### Possible values
+
+| Value | UI value | Description |
+|------|------|------|
+| `userId` | "Assigned only" | Only an assigned user can perform actions on a draft of the Content item. |
+| `null` | "none" | All users can perform actions on all drafts. |
```

Comment:
It is not like that, it is about what *user* can.

Here, user can perform actions regardless of lock existence or who is assigned.